tests/spec/core/biblio-spec.js

```diff
@@ -212,3 +209,23 @@ describe("W3C — Bibliographic References", () => {
     expect(refs[0].textContent).toEqual("[dom]");
   });
 });
+
+it("makes sure references section has expected localization text", async () => {
+  const ops = {
+    config: makeBasicConfig(),
+    htmlAttrs: {
+      lang: "nl",
+    },
+    body: `
+    <section class="informative" id="intro">[[DOM]]</section>
+    <section>[[HTML]]</section>
+    `,
+  };
+  const doc = await makeRSDoc(ops);
+  const { textContent } = doc.querySelector("#references h2");
+  const [normRef, infoRef] = doc.querySelectorAll("#references h3");
+  expect(doc.documentElement.lang).toBe("nl");
+  expect(textContent).toMatch("Referenties");
+  expect(normRef.textContent).toMatch("Normatieve referenties");
```

Comment:
Use `toContain()` instead, as toMatch() is for regular expressions.

Reply:
Oh ok @marcoscaceres. got  it! Would I need to do a squash for my commits?